src/wallet/wallet.cpp

```diff
@@ -2344,47 +2344,32 @@ bool CWalletTx::IsEquivalentTo(const CWalletTx& _tx) const
         return CTransaction(tx1) == CTransaction(tx2);
 }
 
-// Rebroadcast transactions from the wallet. We do this on a random timer
-// to slightly obfuscate which transactions come from our wallet.
-//
-// Ideally, we'd only resend transactions that we think should have been
-// mined in the most recent block. Any transaction that wasn't in the top
-// blockweight of transactions in the mempool shouldn't have been mined,
-// and so is probably just sitting in the mempool waiting to be confirmed.
-// Rebroadcasting does nothing to speed up confirmation and only damages
-// privacy.
+// Once a day, resbumit all wallet transactions to the node,
```

Comment:
typo "resbumit." The previous comment makes a note about rebroadcasting damaging privacy - maybe you could address that in your replacement?

Reply:
hm, so with these changes, this method is now the wallet resubmitting the txn to the mempool, which has minimal implications for rebroadcasting. So now this functionality is not much of a privacy leak at all. from a historic POV it would make sense to explain that difference, but based on the state of the code after the changes, it seems unrelated to say "not a privacy leak"